UnitsNet.Tests/QuantityIFormattableTests.cs

```diff
@@ -65,6 +65,15 @@ public void SFormatEqualsSignificantDigits(string sFormatString, string expected
             Assert.Equal(expected, length.ToString(sFormatString, NumberFormatInfo.InvariantInfo));
         }
 
+        [Fact]
+        public void FormattingUsesSuppliedLocale()
```

Comment:
A short comment would help on why this choice of culture was used. Perhaps that it was to demonstrate a different thousands separator than is used in US English and invariant culture, which many developers also have configured when running tests locally.

Reply:
Shouldn't the test be doing a parse on foot inches?

Reply:
Oh yeah, good point. I thought the generic implementation was fixed, but it was only feetinch parsing. Good catch.

Reply:
This test is actually not directly related to the other issue. This was to check that the specified culture argument really ends up being passed to the `ToString(string, IFormatInfo)` method - and is correctly used there. 

I didn't add new tests for the FootInches stuff, as that test already existed (and failed in the first place when I was initialy building the library).

Reply:
Background: I was initially trying to do `$"{temperature}".ToString(myCulture)`, but for reasons I don't quite understand yet, this didn't work and the Temperature.ToString method would always get null as culture argument.

Reply:
Yes, but we should add tests that show the failure for everyone / Appveyor builds. Pass an explicit culture such as your own.

Reply:
I've clarified the test case. Looks to me as if I ran into a compiler bug :-/ 

@tmilnthorp : Unfortunatelly, I don't fully understand what I should be doing. The build did pass (we'll, before running into a merge issue that is - i'll fix that).

Reply:
@pgrawehr 
> $"{temperature}".ToString(myCulture) 
> but for reasons I don't quite understand yet, this didn't work and the Temperature.ToString method would always get null as culture argument.

Won't the string interpolation call the parameterless `ToString()`, and that's maybe why you are seeing null?

Reply:
That's just calling ToString on a string literal. Temperature was already converted to a string at construction time. Equivalent to temperature.ToString().ToString(myCulture)

Reply:
And a string literal when passed a format provider does nothing

Reply:
https://docs.microsoft.com/en-us/dotnet/api/system.string.tostring?view=netcore-3.1#System_String_ToString_System_IFormatProvider_